<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PringKvitok</class>
 <widget class="QGroupBox" name="PringKvitok">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Печать квитанций</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="displayFormat">
        <string>MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_groups"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_select">
       <property name="text">
        <string>выбрать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_print">
       <property name="text">
        <string>печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_pdf">
       <property name="text">
        <string>pdf</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_duplex">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Если ваш принтер поддерживает печать на 2 стороны&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>На 2 строны</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tW"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
